services/processor/native/call_test.go

```diff
@@ -1,174 +1,192 @@
 package native
 
 import (
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/stretchr/testify/require"
+	"math/big"
 	"reflect"
 	"testing"
 )
 
-func TestPrepareMethodArgumentsForCallWithUint32(t *testing.T) {
-	methodInstance := func(a uint32) uint32 {
-		return a
+func TestPrepareMethodArgumentsAndCall_SimpleOneInputArg(t *testing.T) {
```

Comment:
I assume these new tests cover all the edge cases you handled?

Reply:
the two table tests check only the simple version (one in/one out) as a way to show which types are supported. the more complex tests are for edge cases i could think of.